github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-tekton-tasks/tekton-tasks-periodics.yaml

```diff
@@ -0,0 +1,39 @@
+periodics:
+  - name: periodic-update-libguestfs-container-version
+    cron: 0 2 * * *
+    annotations:
+      testgrid-create-test-group: "false"
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    max_concurrency: 1
+    cluster: kubevirt-prow-control-plane
+    extra_refs:
+    - org: kubevirt
+      repo: kubevirt-tekton-tasks
+      base_ref: main
+    labels:
+      preset-github-credentials: "true"
+    spec:
+      containers:
+      - image: quay.io/kubevirtci/pr-creator:v20230103-9f4e101
+        env:
+        command: [ "/bin/bash", "-ce" ]
+        args:
+        - |
+          tt_dir=$(cd ../kubevirt-tekton-tasks && pwd)
+          # Fetch kubevirt version
+          KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases | \
+            jq '.[] | select(.prerelease==false) | .tag_name' | sort -V | tail -n1 | tr -d '"')
```

Comment:
If you don't want quotes around jq's output, you can specify the `-r` option instead of piping to `tr`. jq has also the `sort` or `sort_by` filter and it is possible to select the last element of an array instead of using tail.